eng/pipelines/templates/stages/archetype-java-release-patch.yml

```diff
@@ -285,6 +286,7 @@ stages:
         - deployment: PublishDocs
           displayName: Publish Docs to GitHubIO Blob Storage
           condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'))
+          timeoutInMinutes: 120
```

Comment:
Are these increases because of spring?

Reply:
No, these increases are because we have about 130 com.azure artifacts that could be released. Right now, the BOM file's contents are limited to the com.azure groupId which cuts out spring and resourcemanager libraries because they have a different groupId. I looked at the time it took to release the 3 template libraries and adjusted things accordingly.

Reply:
No, the BOM is limited to com.azure libraries, and there are around 130 of them (which doesn't include spring or resourcemanager libraries, those have different groupIds and aren't in the BOM). Any/all of which could end up in the patch release. This just a safety net in terms of time. The GitHubIO is way faster than Docs.MS publishing but still takes about 2 minutes to process 3 libraries.